### PR TITLE
Support for the HDArea.co  tracker.

### PR DESCRIPTION
**Pull request by mmarquezs**
I'm not 100% sure with the categories but apart form that it works fine.

### CODE REVIEW

**Comment by lxt2** [comment]
Nice one, have you checked it with `cardigann test definitions/hdarea.yml`?

**Comment by mmarquezs** [comment]
Yes,
```
Definition file parsing OK
INFO[0000] Testing search mode "search"                 
INFO[0001] Successfully logged in                        site=hdarea
INFO[0001] Searching indexer                             query=limit=5&t=search site=hdarea
INFO[0002] Query returned 5 results                      site=hdarea time=468.392436ms
INFO[0002] Testing downloading torrent                   url=http://www.hdarea.co/download.php?id=15106
INFO[0002] Downloaded 85543 bytes from linked torrent   
Indexer test returned OK
```

File: definitions/hdarea.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      selector: table.torrents tbody tr.nonstick_outer_bg
+    fields:
+      category:
+        selector: td:nth-child(1) > a:nth-child(1)
+        attribute: href
+        filters:
+          - name: regexp
+            args: "\\?cat=(\\d+)"
+      title:
+        selector: td:nth-child(2) table.torrentname tbody td a
```
Comment (mmarquezs) [inline definitions/hdarea.yml]: Little mistake, this should be :
```
        selector: td:nth-child(2) table.torrentname tbody td a b
```

**Comment by mmarquezs** [comment]
There is a little bug. I messed up and deleted the fork and one point and I can't edit or add a new commit or at least I can see how (new to pull requests) so I can' fix it without closing the pull request and reopening. It's very small so maybe you can fix it yourself after or if you want I can make a new pull request. 
Here https://github.com/cardigann/cardigann/pull/50/commits/344e295eb6e0bccfb4b096b794eabdc3404ac817#r81450320

**Comment by lxt2** [comment]
Easiest way is to run this in your branch:

```
git fetch  # pull down the latest changes from upstream
git reset --hard origin/master # reset everything back to upstream master
git cherry-pick 344e295eb6e0bccfb4b096b794eabdc3404ac817 # grab just the commit you made
git push -f <branchname> # you need to force push because you have rewritten history somewhat
```

Let me know if you need a hand with this.

**Pull request merged**